drivers/SmartThings/zigbee-contact/src/sengled/init.lua

```diff
@@ -0,0 +1,56 @@
+local clusters = require "st.zigbee.zcl.clusters"
+local battery_defaults = require "st.zigbee.defaults.battery_defaults"
+
+local IASZone = clusters.IASZone
+local PowerConfiguration = clusters.PowerConfiguration
+
+local FINGERPRINTS = {
+  { mfr = "sengled", model = "E2D-G73" }
+}
+
+local CONFIGURATIONS = {
+  {
+    cluster = IASZone.ID,
+    attribute = IASZone.attributes.ZoneStatus.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = IASZone.attributes.ZoneStatus.base_type,
+    reportable_change = 1
+  },
+  {
+    cluster = PowerConfiguration.ID,
+    attribute = PowerConfiguration.attributes.BatteryVoltage.ID,
+    minimum_interval = 30,
+    maximum_interval = 3600,
+    data_type = PowerConfiguration.attributes.BatteryVoltage.base_type,
+    reportable_change = 1
+  }
+}
+
+local is_sengled_products = function(opts, driver, device, ...)
+  for _, fingerprint in ipairs(FINGERPRINTS) do
+    if device:get_manufacturer() == fingerprint.mfr and device:get_model() == fingerprint.model then
+      return true
+    end
+  end
+  return false
+end
+
+local function device_init(driver, device)
+  battery_defaults.build_linear_voltage_init(2.0, 3.0)(driver, device)
```

Comment:
If you use this method, the battery voltage attribute will be configured for you using the defaults. The only difference between your config and the defaults is the max reporting interval, which is 21600 in the defaults.

Similarly, your IAS Zone configuration is not that far removed from the defaults (a lower max interval of 300) and a `nil` reportable_change.

I think, if you wanted, all you'd have to have is the `linear_voltage_init`. You could use tests to confirm that the results of the configure lifecycle event are what you expect.